README.adoc

```diff
@@ -50,6 +50,232 @@ with Spring Cloud Netflix provides Intelligent Routing (Zuul), Client Side Load
 :sc-ext: java
 :project-full-name: Spring Cloud Zookeeper
 
+== Quick Start
+
+This quick start walks through using Spring Cloud Zookeeper for Service Discovery and Distributed Configuration.
+
+First, run Zookeeper on your machine, then you can access it and use as Service Registry and Configuration source with Spring Cloud Zookeeper.
```

Comment:
Change to:

First, run Zookeeper on your machine. Then you can access it and use it as a Service Registry and Configuration source with Spring Cloud Zookeeper.